hack/update-protobuf.sh

```diff
@@ -16,13 +16,12 @@ fi
 
 rm -rf go-to-protobuf
 rm -rf protoc-gen-gogo
-go build -o _output/bin/go-to-protobuf github.com/openshift/api/vendor/k8s.io/code-generator/cmd/go-to-protobuf
-go build -o _output/bin/protoc-gen-gogo github.com/openshift/api/vendor/k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo
+go build -o _output/bin/go-to-protobuf ./vendor/k8s.io/code-generator/cmd/go-to-protobuf
+go build -o _output/bin/protoc-gen-gogo ./vendor/k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo
 
 PATH="$PATH:_output/bin" go-to-protobuf \
   --output-base="${GOPATH}/src" \
   --apimachinery-packages='-k8s.io/apimachinery/pkg/util/intstr,-k8s.io/apimachinery/pkg/api/resource,-k8s.io/apimachinery/pkg/runtime/schema,-k8s.io/apimachinery/pkg/runtime,-k8s.io/apimachinery/pkg/apis/meta/v1,-k8s.io/apimachinery/pkg/apis/meta/v1beta1,-k8s.io/api/core/v1,-k8s.io/api/rbac/v1' \
   --go-header-file=${SCRIPT_ROOT}/hack/empty.txt \
-  --proto-import=${SCRIPT_ROOT}/vendor/github.com/gogo/protobuf/protobuf \
```

Comment:
this breaks the script:

```
github.com/openshift/api/apps/v1/generated.proto: Import "github.com/gogo/protobuf/gogoproto/gogo.proto" was not found or had errors.
github.com/openshift/api/apps/v1/generated.proto: warning: Import k8s.io/apimachinery/pkg/runtime/schema/generated.proto but not used.
2019/12/13 13:36:09 protoc -I . -I /Users/sts/Quellen/origin/src -I hack/../vendor --gogo_out=/Users/sts/Quellen/origin/src /Users/sts/Quellen/origin/src/github.com/openshift/api/apps/v1/generated.proto
2019/12/13 13:36:09 Unable to generate protoc on github.com.openshift.api.apps.v1: exit status 1
```

Reply:
@sttts you have incomplete protobuf installation on your machine. Try the version from https://github.com/google/protobuf/releases.